examples/electron/RunAnywhereAI/README.md

```
@@ -1,153 +1,63 @@
-| **Repo checkout** | Full monorepo clone |
-| **Optional GPU build** | CUDA prebuild at `prebuilds/win32-x64-cuda/` (see SDK README) |
-
----
-
-## Setup
-
-### 1. Clone the repository
-
-```bash
-git clone https://github.com/RunanywhereAI/runanywhere-sdks.git
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)